admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1392,6 +1392,44 @@ $product-page-padding-bottom: 80px !default;
   }
 }
 
+.product_packed_products {
+  .search-with-icon {
+    max-width: 377px;
+  }
+
+  .entities-list-container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-right: -15px;
+    margin-left: -15px;
```

Comment:
```suggestion
    margin-right: -0.5rem;
    margin-left: -0.5rem;
```

0.5rem = 8px, not 15

Because the container is overflowing now:
![image](https://user-images.githubusercontent.com/14963751/169252724-370ccd44-4532-45c7-9c45-4ad1787fc687.png)

Reply:
@NeOMakinG the UX is not final we have planned to do a full review of the page with @TristanLDD and @Julievrz after the feature freeze, but it's not worth blocking the whole feature for this as it is a known issue

The feature needs to be merged before feature freeze, I don't think UX glitches, which are planned to be fixed during feature freeze, should block this feature and mean we don't release the product page at all

Reply:
If your change is enough to fix it though it's worth doing this mini change, but if it's a lot of work to fix all the UX bugs then it should be done in another dedicated PR

Reply:
This change is enough to fix a glitch that we obviously don't want

I'm not blocking this feature because the UI is ugly, I'm blocking it because it contains a UI bug that could probably cause an overflowing div on mobile

Reply:
I apply the requested change